README.md

```diff
@@ -30,16 +30,15 @@
     source ENV/bin/activate
     ```
 
-5. Navigate to the arches-rdm package, and install it into the virtual environment:
+5. Navigate to the `arches-lingo` directory, and install the project (with development dependencies):
     ```
-    cd arches
-    pip install -e .
+    cd arches-lingo
+    pip install -e '.[dev]'
     ```
 
-6. Navigate to the arches_lingo, and install the python requirements:
+6. Also install core arches for local development:
     ```
-    cd ../arches-lingo
-    pip install -r arches_lingo/install/requirements.txt
+    pip install -e ../arches
```

Comment:
I reversed the order of these commands because doing it in the other order will undo your editable install of arches.